src/models/GenericModel.ts

```
@@ -0,0 +1,12 @@
+// GenericModel class
+export default abstract class GenericModel {
+    // Abstract property for table name
+    static TABLE_NAME: string;
+
+    static DATABASE_MAP: Object;
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)